lambda/matching.ml

```diff
@@ -1961,19 +1983,20 @@ let get_expr_args_constr ~scopes head arg rem =
       else
         {
           arg = Lprim (Pfield (pos, Pointer, Immutable), [ arg ], loc);
+          mut;
```

Comment:
Should we also multiply by 1 here? (There are also the inlined record / unboxed variant cases below, but I would say that we shouldn't multiply by 1 there, as there's not actually a "read" happening at runtime.)

Reply:
Indeed, this was an omission on my part, thanks! I also tweaked the code of `get_expr_args_record` to make it more visually similar to the other.